src/Raven.Server/RavenServerStartup.cs

```diff
@@ -316,7 +316,7 @@ private static void MaybeSetExceptionStatusCode(HttpContext httpContext, Excepti
 
             if (exception is DatabaseNotRelevantException)
             {
-                response.StatusCode = (int)HttpStatusCode.Gone;
+                response.StatusCode = (int)HttpStatusCode.SeeOther;
```

Comment:
This is meant to be used with a `Location` header, and that isn't a good idea, we aren't sending it.
What is better for us is to actually specify the header values so this isn't cached.